ITR/data/excel.py

```diff
@@ -39,106 +29,167 @@ def _check_sector_data(self) -> None:
         :return: None
         """
         assert pd.Series([TabsConfig.PROJECTED_PRODUCTION, TabsConfig.PROJECTED_EI]).isin(
-            self.sector_data.keys()).all(), "some tabs are missing in the sector data excel"
+            self.benchmark_excel.keys()).all(), "some tabs are missing in the sector data excel"
 
-    def get_targets(self, company_ids: List[str]) -> List[IDataProviderTarget]:
+    def get_company_projected_production(self, ghg_scope12: pd.DataFrame) -> pd.DataFrame:
         """
-        Get all relevant targets for a list of company ids (ISIN). This method should return a list of
-        IDataProviderTarget instances.
+        get the projected productions for list of companies in ghg_scope12
+        :param ghg_scope12: DataFrame with at least the following columns :
+        ColumnsConfig.COMPANY_ID,ColumnsConfig.GHG_SCOPE12, ColumnsConfig.SECTOR and ColumnsConfig.REGION
+        :return: DataFrame of projected productions for [base_year - base_year + 50]
+        """
+        benchmark_production_projections = self.get_benchmark_projections(ghg_scope12)
+        return benchmark_production_projections.add(1).cumprod(axis=1).mul(
+            ghg_scope12[self.column_config.GHG_SCOPE12].values, axis=0)
 
-        :param company_ids: A list of company IDs (ISINs)
-        :return: A list containing the targets
+    def get_benchmark_projections(self, company_sector_region_info: pd.DataFrame) -> pd.DataFrame:
         """
-        model_targets = self._target_df_to_model(self.company_data['target_data'])
-        model_targets = [target for target in model_targets if target.company_id in company_ids]
-        return model_targets
+        Overrides subclass method
+        returns a Dataframe with intensity benchmarks per company_id given a region and sector.
+        :param company_sector_region_info: DataFrame with at least the following columns :
+        ColumnsConfig.COMPANY_ID, ColumnsConfig.SECTOR and ColumnsConfig.REGION
+        :return: A DataFrame with company and intensity benchmarks per calendar year per row
+        """
+        return self.get_benchmark_projections_from_sheet(company_sector_region_info, TabsConfig.PROJECTED_PRODUCTION)
 
-    def _target_df_to_model(self, df_targets: pd.DataFrame) -> List[IDataProviderTarget]:
+    def get_benchmark_projections_from_sheet(self, company_sector_region_info: pd.DataFrame,
+                                             feature: str) -> pd.DataFrame:
```

Comment:
Possible to remove argument 'feature', since this function is only called from within this class => feature == TabsConfig.PROJECTED_PRODUCTION